azure-pipelines-e2e-tests-template.yml

```diff
@@ -107,45 +113,48 @@ stages:
             SPARK_HOME: $(Build.BinariesDirectory)$(PATH_SEPARATOR)spark-${{ test.version }}-bin-hadoop2.7
             DOTNET_WORKER_DIR: $(CURRENT_DOTNET_WORKER_DIR)
 
-        # - pwsh: |
-        #     echo "Downloading ${env:BACKWARD_COMPATIBLE_WORKER_URL}"
-        #     curl -k -L -o Microsoft.Spark.Worker-${{ parameters.backwardCompatibleRelease }}.zip ${env:BACKWARD_COMPATIBLE_WORKER_URL}
-        #     unzip Microsoft.Spark.Worker-${{ parameters.backwardCompatibleRelease }}.zip -d $([System.IO.Directory]::GetParent($env:BACKWARD_COMPATIBLE_DOTNET_WORKER_DIR).FullName)
-
-        #     if ($env:AGENT_OS -eq 'Linux') {
-        #       chmod +x "${env:BACKWARD_COMPATIBLE_DOTNET_WORKER_DIR}${env:PATH_SEPARATOR}Microsoft.Spark.Worker"
-        #     }
-        #   displayName: 'Setup Backward Compatible Microsoft Spark Worker ${{ parameters.backwardCompatibleRelease }}'
-        #   workingDirectory: $(Build.BinariesDirectory)
-
-        # - task: DotNetCoreCLI@2
-        #   displayName: 'E2E Backward Compatibility Tests'
-        #   inputs:
-        #     command: test
-        #     projects: '**/Microsoft.Spark*.E2ETest/*.csproj'
-        #     arguments: '--configuration $(buildConfiguration) ${{ option.backwardCompatibleTestOptions }}'
-        #     workingDirectory: $(Build.SourcesDirectory)$(PATH_SEPARATOR)dotnet-spark
-        #   env:
-        #     HADOOP_HOME: $(Build.BinariesDirectory)$(PATH_SEPARATOR)hadoop
-        #     SPARK_HOME: $(Build.BinariesDirectory)$(PATH_SEPARATOR)spark-${{ test.version }}-bin-hadoop2.7
-        #     DOTNET_WORKER_DIR: $(BACKWARD_COMPATIBLE_DOTNET_WORKER_DIR)
+        - pwsh: |
+             echo "Downloading ${env:BACKWARD_COMPATIBLE_WORKER_URL}"
+             curl -k -L -o Microsoft.Spark.Worker-${{ parameters.backwardCompatibleRelease }}.zip ${env:BACKWARD_COMPATIBLE_WORKER_URL}
+             unzip Microsoft.Spark.Worker-${{ parameters.backwardCompatibleRelease }}.zip -d $([System.IO.Directory]::GetParent($env:BACKWARD_COMPATIBLE_DOTNET_WORKER_DIR).FullName)
+
+             if ($env:AGENT_OS -eq 'Linux') {
+               chmod +x "${env:BACKWARD_COMPATIBLE_DOTNET_WORKER_DIR}${env:PATH_SEPARATOR}Microsoft.Spark.Worker"
+             }
+          displayName: 'Setup Backward Compatible Microsoft Spark Worker ${{ parameters.backwardCompatibleRelease }}'
+          workingDirectory: $(Build.BinariesDirectory)
+
+        - task: DotNetCoreCLI@2
+          displayName: 'E2E Backward Compatibility Tests'
+          inputs:
+             command: test
```

Comment:
indentation looks off.  Should be 2 spaces?  Can you fix this for the other places too?  I think indentation is off for all the lines you uncommented.

Reply:
Fixed.